src/getting-started/choose-web-library.md

```diff
@@ -115,7 +115,6 @@ There are a few different angles to consider when choosing between using `web-sy
       </td>
     </tr>
   </tbody>
-</table>For more information about picking a build tool, checkout the following page:
-
-
+</table>
 
+For more information about picking a build tool, checkout the following page:
```

Comment:
I noticed that there is a link below in the document website, but it is not shown here. Why? Is it controlled by this non-newline code?

Reply:
Sometimes Gitbook does strange things that I don't understand 😅 Anyways, the link on Gitbook was wrong so I just updated it to be an inline link so this shouldn't be an issue anymore. Can you rebase?